presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergAbstractMetadata.java

```diff
@@ -322,4 +335,101 @@ public void rollback()
     {
         // TODO: cleanup open transaction
     }
+
+    @Override
+    public void addColumn(ConnectorSession session, ConnectorTableHandle tableHandle, ColumnMetadata column)
+    {
+        if (!column.isNullable()) {
+            throw new PrestoException(NOT_SUPPORTED, "This connector does not support add column with non null");
+        }
+        IcebergTableHandle handle = (IcebergTableHandle) tableHandle;
+        org.apache.iceberg.Table icebergTable = getIcebergTable(session, handle.getSchemaTableName());
```

Comment:
nit: Since we have already imported, we can use Table instead of the qualified name here as well as at the other occurrences of the same.

Reply:
fixed